### ISSUE TITLE
Android Basics: Activities and intents

### ISSUE THREAD

**Issue by lucascsoregh** [google-developer-training/android-basics-kotlin-words-app#303]
**[URL of codelab](https://developer.android.com/codelabs/basic-android-kotlin-training-activities-intents?continue=https%3A%2F%2Fdeveloper.android.com%2Fcourses%2Fpathways%2Fandroid-basics-kotlin-unit-3-pathway-1%23codelab-https%3A%2F%2Fdeveloper.android.com%2Fcodelabs%2Fbasic-android-kotlin-training-activities-intents#2)**

**In which task and step of the codelab can this issue be found?**
All of it

**Describe the problem**
Basically the entire codelab is broken because the github code does not match up with the text in the codelab at all.

**Steps to reproduce?**
1. Go to...
2. Click on...
3. See error...

**Versions**
_Android Studio version:_ 
_API version of the emulator:_ 

**Additional information**
_Include screenshots if they would be useful in clarifying the problem._
